test/unit/io_clock.js

```diff
@@ -0,0 +1,34 @@
+var test = require('tap').test;
+var Clock = require('../../src/io/clock');
+var Runtime = require('../../src/engine/runtime');
+
+test('spec', function (t) {
+    var rt = new Runtime();
+    var c = new Clock(rt);
+
+    t.type(Clock, 'function');
+    t.type(c, 'object');
+    t.type(c.projectTimer, 'function');
+    t.type(c.pause, 'function');
+    t.type(c.resume, 'function');
+    t.type(c.resetProjectTimer, 'function');
+    t.end();
+});
+
+test('cycle', function (t) {
+    var rt = new Runtime();
+    var c = new Clock(rt);
+
+    t.ok(c.projectTimer() <= 0.001);
```

Comment:
is it possible the test could fail if we take a lot of time between instantiation of `c` and this line here? Seems exceedingly unlikely, but could it happen in a slow-CPU environment like travis?